broker_test.go

```diff
@@ -1,6 +1,53 @@
 package main
 
-import "testing"
+import (
+	"os"
+	"testing"
+	"time"
+
+	"github.com/hashicorp/vault/command"
+)
+
+const testToken = "root"
+
+func TestIntegration(t *testing.T) {
+	if err := setup(); err != nil {
+		t.Fatal(err)
```

Comment:
Why even have setup() return an err?  What would any test do other than Fatal in that scenario?  If nothing, why not move the Fatal into setup()?